feature-libs/my-account/organization/components/cost-center/services/assign.service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+import { CurrentOrganizationItemService } from '../../shared/current-organization-item.service';
+
+@Injectable()
+export abstract class OrganizationAssignService<T> {
```

Comment:
This service seems not being used anymore. It's file name doesn't match the class name. Can we remove it?

Reply:
Removed that one